packages/lib/src/vis/surface/SurfaceMesh.tsx

```diff
@@ -20,30 +21,28 @@ function SurfaceMesh(props: Props) {
   const [dataGeometry] = useState(() => new BufferGeometry());
   const invalidate = useThree((state) => state.invalidate);
 
-  const dataToColorScale = useDataToColorScale(
-    scaleType,
-    domain,
-    colorMap,
-    invertColorMap
-  );
+  const safeDataArray = useTextureSafeNdArray(dataArray);
 
-  const {
-    position,
-    color,
-    index: indices,
-  } = useBufferAttributes(dataArray, dataToColorScale);
+  const { position, index: indices, uv } = useBufferAttributes(dataArray);
```

Comment:
Now that the returned values are of type `BufferAttribute`, it might make more sense to keep the name `index` as is instead of renaming it.

Reply:
You are right. I don't know why there was still this back-and-forth